pyproject.toml

```diff
@@ -3,31 +3,28 @@ name = "argowrapper"
 version = "2.2.0"
 description = "argo wrapper for va workflow"
 authors = ["UchicagoZchen138 <zchen138@uchicago.edu>"]
+readme = "README.md"
 repository = "https://github.com/uc-cdis/argo-wrapper"
 
 [tool.poetry.dependencies]
 python = "^3.9"
-fastapi = "^0.71.0"
-uvicorn = "^0.16.0"
-gunicorn = "^20.0.4"
-importlib-metadata = "^4.10.0"
-PyYAML = "6.0.1"
-pytest = "^6.2.5"
-pytest-mock = "^3.6.1"
-pytest-asyncio = "^0.20"
+argo-workflows = "^6.5"
 cdislogging = "^1.1.1"
-argo-workflows = "6.4.7"
+fastapi = "^0"
```

Comment:
Just curious what's the reason for pinning fastapi to a lower version than before?

Reply:
I was thinking of keeping it to [`^0`](https://python-poetry.org/docs/dependency-specification/#caret-requirements), so we're forced to update FastAPI on new versions (to keep up with them and any security updates), but if it raises questions then I'll pin it to [0.115 as recommended](https://fastapi.tiangolo.com/deployment/versions/#pin-your-fastapi-version).